sidebars.js

```diff
@@ -87,6 +87,11 @@ module.exports = {
         "type": "doc",
         "id": "mlapi-basics/networkobject"
```

Comment:
FWIW, I still highly suggest doing a scrub and getting rid of "mlapi" routes/directories/links for the 1.0 release and going forward. It is probably the safest and the most headacheless (is that a word?) time to do this IMHO.